coordinator/storage/mocks/StorageDriver.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2015 The Serviced Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mocks
+
+import "github.com/stretchr/testify/mock"
+
+type StorageDriver struct {
+	mock.Mock
+}
+
```

Comment:
Actually, coordinator/storage/monitor_test.go defines a MockStorageDriver already. 
Is there a way to share one mock across tests in two different packages?